src/Command/Customer/UpdateCustomer.php

```diff
@@ -55,7 +56,7 @@ public function email(): string
         return $this->email;
     }
 
-    public function birthday(): ?string
+    public function birthday(): ?\DateTimeImmutable
```

Comment:
```suggestion
    public function birthday(): ?DateTimeImmutable
```
as it is already imported